### PR TITLE
Newtest

### PR DESCRIPTION
**Pull request by Saghana97**
New request.

### CODE REVIEW

File: Newfolder_testing/access.js [hunk 1]
```diff
@@ -1 +1,3 @@
-hello.js
```
Comment (Saghana97) [inline Newfolder_testing/access.js]: New test

File: Newfolder_testing/access.js [hunk 2]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+hello.js
+
```
Comment (Saghana97) [inline Newfolder_testing/access.js]: Testing api

**Comment by anish-codingmart** [comment]
Testing

File: Newfolder_testing/access.js (hunk 2 above)
Comment (Saghana97) [reply Newfolder_testing/access.js]: lll

Comment (Saghana97) [reply Newfolder_testing/access.js]: poi

Comment (Saghana97) [reply Newfolder_testing/access.js]: api test

File: Newfolder_testing/access.js [hunk 3]
```diff
@@ -1 +1,3 @@
-hello.js
\ No newline at end of file
+hello.js
```
Comment (anish-codingmart) [inline Newfolder_testing/access.js]: Test

Comment (Saghana97) [reply Newfolder_testing/access.js]: reply for test

**Comment by anish-codingmart** [comment]
Testing Comments now!

File: Newfolder_testing/access.js (hunk 1 above)
Comment (Saghana97) [reply Newfolder_testing/access.js]: fdsfdsfds

**Pull request closed without merging**